ForgeUnity/Assets/BeardedManStudios/Scripts/NetworkManager.cs

```diff
@@ -61,13 +63,19 @@ protected virtual void Awake()
 		protected virtual void OnEnable()
 		{
 			if (automaticScenes)
+			{
 				SceneManager.sceneLoaded += SceneReady;
+				SceneManager.sceneUnloaded += SceneUnloaded;
```

Comment:
Might be worth renaming `SceneReady` to `SceneLoaded` to make the naming consistent? :thinking: